src/main/cpp/test/TPImpeller.cpp

```diff
@@ -0,0 +1,34 @@
+#include <test/TPImpeller.h>
+#include <controllers/impeller/ImpellerStateMgr.h>
+#include <controllers/impeller/ImpellerOn.h>
+#include <controllers/impeller/ImpellerOff.h>
+#include <controllers/impeller/ImpellerAgitate.h>
+using namespace frc;
+
+TPImpeller::TPImpeller()
+{
+    m_timer = new Timer;
+
+}
+
+void TPImpeller::periodic()
+{
+    auto time = m_timer -> Get();
+    if (time < 15)
+    {
+        SetCurrentState(ImpellerStateMgr::IMPELLER_STATE::ON, true);
```

Comment:
should be m_impellerMgr-> or whatever you call your attribute

this means you do need the impellerstatemgr included here